### ISSUE TITLE
Add support for private repositories

### ISSUE THREAD

**Issue by trajano** [mcuadros/ofelia#178]
In order to avoid breaking the Docker Hub limits it would be nice to allow using private proxy repositories.

**Comment by maietta** [comment]
Agreed. I run a fork of Ofelia called Chadburn and have implemented Github Container Registry in addition to Docker Hub for this very reason. If this might help you, the project is located at https://github.com/PremoWeb/Chadburn.

**Comment by trajano** [comment]
Thanks @maietta it should.  There's a few more problems I'm dealing with, if you're actively working it I'll open the issues there.
